src/test/java/pl/sda/shop/onlineshop/service/CategoryServiceTest.java

```diff
@@ -9,21 +9,27 @@
 import pl.sda.shop.onlineshop.model.Category;
 import pl.sda.shop.onlineshop.repository.CategoryRepository;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
 import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 
 @ExtendWith(MockitoExtension.class)
 class CategoryServiceTest {
-    private static final Category CATEGORY = new Category(1L, "Computers");
+    private static final Category CATEGORY = new Category(1L, "Computers", null);
+    private static final Category CATEGORY2 = new Category(2L, "Notebooks", CATEGORY);
+
+    List<Category> categories = Arrays.asList(CATEGORY, CATEGORY2);
```

Comment:
jeżeli to stała to z dużych liter, brakuje też modyfikatora dostępu, static i final

Reply:
Poprawione.